packages/core/src/components/info/info.scss

```diff
@@ -1,19 +1,21 @@
 @use "~dso-toolkit/src/utilities";
-@use "~dso-toolkit/src/components/button";
-@use "~dso-toolkit/src/components/info";
+@use "~dso-toolkit/src/components/button/button.mixins" as button-mixins;
+@use "~dso-toolkit/src/components/info/info.mixins" as info-mixins;
```

Comment:
Sorry, ik was niet duidelijk in mijn vorige comment. Als componenten onderling importeren, verwijzen we naar de `_index.scss` file.

In dit geval wilt het Web Component Info component mixins importeren van het component button en info. Zie ook mijn hippe afbeelding en het comment uit een ander PR: https://github.com/dso-toolkit/dso-toolkit/pull/2662/files#diff-3fd10bd0e8cf5b660413ccc0fcf1d2ffac77388404d4ca126800a0837e6ed270.

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/a9deb45f-6a78-438d-aad6-3b8a73a17f16)

Ofwel: De change op deze twee regels terugdraaien en importeren en respectievelijk uit de `button` en `info` namespaces includen.

Reply:
Helder.

Reply:
Aangepast

Reply:
Check

Reply:
In een ander comment zeg je dat je dit hebt teruggedraaid, echter zie ik hier nog steeds een change die ik niet begrijp (en niet conform de afbeelding is).

Mis ik iets? 🤔

Reply:
Check! 😄